README.md

```diff
@@ -908,76 +908,101 @@ with ```sudo```. Please see the linux-postinstall page in the Docker documentati
+cp emissary-knight.png target/data/InputData/
 ```
 
-Once Emissary starts up, we should see a log line that says: "Started EmissaryServer at http://localhost:8001." We now can copy files into 
-```input/InputData/``` for Emissary to process. When the processing has finish, the files will be moved to ```input/DoneData```. All extracted 
-content can be found in the local ```output``` directory. Depending on your specific configuration, there should be output in the ```output/json``` 
-directory.
+When the processing has finish, the files will be moved to ```target/data/DoneData```. All extracted content can be 
+found in the local ```target/localoutput``` directory. If using the default Emissary configuration, there should be 
+output in the ```target/localoutput/json``` directory. Output should look similar to:
+
+```
+[emissary]$ ls target/localoutput/json/
+202102281211000localhost.json_1ddf101f-0c32-4b10-ba14-da3521c8ec68.bgpart
+202102281211000localhost.json_1ddf101f-0c32-4b10-ba14-da3521c8ec68.bgpart.bgjournal
+```
 
-To monitor Emissary in a container, we need to connect to the running container. If we want to run the agents command,
+To monitor Emissary, we need to connect to the running container. If we want to run the agents command,
 we simply need to run the following command:
+
 ```
 docker exec -it emissary ./emissary agents -p 8001 --mon
 ```
 
 To see the environment settings:
+
 ```
 docker exec -it emissary ./emissary env
 ```
 
 If you want to connect to the web front-end for Emissary, some additional steps are need to configure jetty. We need to
 give the container a hostname and pass that onto Emissary from the command line. Sample command:
+
 ```
 docker run -it --rm --name emissary --hostname emissary-001 -p 8001:8001 emissary server -a 2 -p 8001 -s http -h emissary-001
 ```
 
 Then from a browser, assuming container is running locally, go to http://localhost:8001/ to see the endpoints.
 
 ### Cluster Mode using Docker Compose
-We can use a Docker compose file to simulate cluster mode. We'll start a feeder and two workers by default. To start the cluster, run the 
-sample docker-compose.yml file:
+We can use a Docker compose file to simulate cluster mode. We'll start a feeder and two workers by default. To start the
+cluster, run the sample docker-compose.yml file. From the root of the project, run:
+
 ```
 docker-compose -f contrib/docker/docker-compose.yml up
 ```
 
-Once the cluster has started, we can manage input and output by looking at the volumes that were created by the compose file. The volume
- location can be found in ```/var/lib/docker/volumes/```. Simply move a file into the input directory for Emissary to process it:
+Use docker copy to run a file through Emissary:
+
 ```
-cp emissary-knight.png /var/lib/docker/volumes/docker_input/_data/InputData/
+docker cp emissary-knight.png docker_emissary-feeder_1:/opt/emissary/target/data/InputData/
```

Comment:
This worked for me! Also, I was able to connect to the http endpoint. Not sure if I did something different or what happened there

This all checks out and I will approve, but I have two desired changes:
1) For http endpoint please add a line with what the username and password for the endpoint are or where to find that info
2) Explain where the emissary-knight.png goes (I couldn't seem to find the log or output data in the normal locations I would expect)